packages/entrypoint/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@matanuska/entrypoint",
+  "version": "1.0.0",
+  "description": "Matanuska's entry point generator",
+  "main": "dist/index.js",
+  "bin": {
+    "entrypoint": "dist/main.js"
+  },
+  "scripts": {
+    "install": "npm run install:terraform",
```

Comment:
This hook doesn't run on `npm install`, it seems. I had to do `npm run install` to trigger this script.

A problem for another day, though.